app/config/parameters.yml.dist

```diff
@@ -24,6 +25,7 @@ parameters:
     sylius.phpcr.backend:
         type: doctrinedbal
         connection: default
+        check_login_on_server: false
```

Comment:
Related issue is https://github.com/doctrine/DoctrinePHPCRBundle/issues/195 but maybe I don't need it anymore. Let me check.